### ISSUE TITLE
EV3 and NXT: Arrow keys cause unexpected behaviour

### ISSUE THREAD

**Issue by Caffetaria** [OpenRoberta/openroberta-lab#612]
**Describe the bug**
 - The left and right arrow keys make the robot turn left and right respectively. This is expected.
The expected behaviour for the up and down arrow keys would be to make the robot move front and back respectively.
However, the up and down arrow keys make the robot turn left and right respectively( similar to the left and right keys).

**To Reproduce**
1. Create a new EV3 or NXT program
2. Open the simulation
3. Test the behaviour of the arrow keys

**Expected behavior**
 - The up and down arrow keys should make the robot move front and back respectively.

**Screenshots**
 - Left Arrow key
![gif16](https://user-images.githubusercontent.com/9400738/72711678-7951fd80-3b8f-11ea-9d6c-64b4e47c6a47.gif)

- Right Arrow Key
![gif15](https://user-images.githubusercontent.com/9400738/72711683-7c4cee00-3b8f-11ea-9ae1-9b5a4a616a22.gif)

- Up Arrow Key
![gif16](https://user-images.githubusercontent.com/9400738/72711678-7951fd80-3b8f-11ea-9d6c-64b4e47c6a47.gif)

- Down Arrow Key
![gif15](https://user-images.githubusercontent.com/9400738/72711683-7c4cee00-3b8f-11ea-9ae1-9b5a4a616a22.gif)

**Device information**
 - Type: Desktop
 - OS: MacOS
 - Browser: Chrome

**Comment by KevinL10** [comment]
I have been able to prevent the up and down keys from turning, but cannot find a way to make the robot move forward. Here's the current progress, someone else can build on this.
![handlekeyevent](https://user-images.githubusercontent.com/58920989/73001828-e942cc80-3db7-11ea-8434-1f19e952339a.PNG)

**Comment by Caffetaria** [comment]
Cool @KevinL10! 
I think I got it working. 
Can I make a pull request?

**Comment by Caffetaria** [comment]
@KevinL10 I believe that it may require some translation to the global coordinate system ... but I could be wrong.

**Comment by Caffetaria** [comment]
@KevinL10 
Here's where I am at the moment. 
Doing something like this works in moving the robot back on hitting the down arrow:

```
case "ArrowDown":
                robots[robotIndex].pose.x -= 1;
                robots[robotIndex].pose.theta -= Math.PI / 180;
```
However, this does not work if the robot is not oriented horizontally. The change in theta is required in order to correct for a 'drift' that occurs in the orientation of the robot. 

Will update if I learn more.

**Comment by boonto** [comment]
Resolved in d7afa3a58686cbcfb368eb296f51dd448b114d04.
